linux_docker_resources/Dockerfile

```diff
@@ -144,7 +145,7 @@ RUN if test ${COMPILE_WITH_CLANG} = true; then apt-get update && apt-get install
 # Install coverage build dependencies.
 RUN apt-get update && apt-get install --no-install-recommends -y lcov
 
-RUN pip3 install -U lcov_cobertura_fix
+RUN pip3 install -U lcov_cobertura && rm /usr/local/lib/python3.12/dist-packages/lcov_cobertura/lcov_cobertura.py && wget https://raw.githubusercontent.com/ahcorde/lcov-to-cobertura-xml/master/lcov_cobertura/lcov_cobertura.py -P /usr/local/lib/python3.12/dist-packages/lcov_cobertura/
```

Comment:
I think we can move forward with a fork of this fix for now, but we should do that from an repository.  We should also add in a comment here explaining exactly why we are doing that.

Let me discuss today with some people to decide where we should put this fork for now.

Reply:
OK, I talked to @nuclearsandwich about this, and I got a much better idea of what we should do here.  I think the easiest thing is to just use your fork for now, since you already have a fork.  In that case, I think this should be:

```suggestion
# There is a bug in upstream lcov-to-cobertura-xml with newer versions of lcov.  There is an
# open PR for it in https://github.com/eriwen/lcov-to-cobertura-xml/pull/55, but until that is
# merged use the fork with the fix.
RUN pip3 install -U git+https://github.com/ahcorde/lcov-to-cobertura-xml@master
```